controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -264,14 +310,21 @@ func (r *OCSInitializationReconciler) Reconcile(ctx context.Context, request rec
 
 // SetupWithManager sets up a controller with a manager
 func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	cluster, err := cluster.New(mgr.GetConfig(), func(options *cluster.Options) {
+		options.Scheme = mgr.GetScheme()
+	})
```

Comment:
@umangachapagain it's been sometime that I looked into controller-runtime (during ODFMS), pls excuse if I'm a bit wrong here.

You are using only Scheme from manager but not the cluster which is auto created and has a cache attached to it. The controller that is being saved on reconciler could probably be configured against managers' cluster cache but may not be against this newly created cluster cache.

If above is correct, we may want to use a single cache across all controllers?